src/patterns/organisms/sidebar-menu/sidebar-menu.hbs

```diff
@@ -0,0 +1,39 @@
+---
+name: Sidebar menu
+description: |
+  Local navigation for a subset of pages or section of a website, intended to appear in a sidebar adjacent to the main content. It can accommodate multiple categories/sub-sections in a two-level hierarchy with titles corresponding to the top level page of each sub-section. See [the article demo page](/demos/article.html) for an example in better context.
+notes: |
+  The menu summary reflects the current page, shown when the menu is collapsed in small viewports. The menu label should be the title of the site section.
+tips: |
+  - Indicate the current page with the `mzp-is-current` class.
+labels:
+    - prototype
+    - inprogress
+---
+
+<nav class="mzp-c-sidemenu">
+  <section class="mzp-c-sidemenu-summary mzp-js-toggle" aria-role="button" aria-controls="">
```

Comment:
I think the value for `aria-controls` should be the ID of the menu?

Reply:
Should both this and `aria-role="button"` be set in the JS along with where you set `tabindex`? They probably don't apply when JS is disabled?